dev.js

```diff
@@ -34,7 +34,7 @@ var loadClientBundleLink = path.join(dirs.meteor, 'client/loadClientBundle.html'
 
 var requireServerBundleJs = path.join(dirs.meteor, 'server/require.server.bundle.js');
 
-require('./core-js-custom-build');
+exec('node core-js-custom-build.js');
 
```

Comment:
does `require` not work here for some reason?  I'd rather use `require` if possible because shelljs claims that synchronous `exec` can have performance problems for long-running processes.  Probably not a big deal here but just makes me want to avoid synchronous exec if possible